dags/analyzer_helper/discourse/transform_raw_members.py

```diff
@@ -0,0 +1,36 @@
+import logging
+
+from dags.analyzer_helper.common.base.transform_raw_members_base import TransformRawMembersBase
+
+
+class TransformRawMembers(TransformRawMembersBase):
+    def transform(self, raw_members: list) -> list:
+        """
+        Transform extracted raw members data into the guildmember structure.
```

Comment:
The docstring is wrong. We're converting discourse member data to the general raw members data structure.